cmd/nginx-ingress/main_test.go

```diff
@@ -22,3 +23,77 @@ func TestValidateStatusPort(t *testing.T) {
 	}
 
 }
+
+func TestParseNginxStatusAllowCIDRs(t *testing.T) {
+
+	var tests = []struct {
+		input         string
+		expected      []string
+		expectedError error
+	}{
+		{"earth, ,,furball",
```

Comment:
A change like below could improve readability.
```
		{"earth, ,,furball",
			[]string{},
			errors.New("invalid IP address: earth")},
```
->
```
		{
			"earth, ,,furball",
			[]string{},
			errors.New("invalid IP address: earth"),
		},
```
Could you possibly reformat `tests` similar to the example above?